lib/pbench/server/api/__init__.py

```diff
@@ -134,7 +134,7 @@ def create_app(server_config):
     register_endpoints(api, app, server_config)
 
     try:
-        Database.init_db(server_config=server_config, logger=app.logger)
+        init_db(server_config=server_config, logger=app.logger)
```

Comment:
How does this work?  `init_db` does not appear to accept keyword arguments, while `Database.init_db` does.

Reply:
What do you mean? they both accept 2 parameters `server_config` and `logger`

Reply:
And, it's an excellent idea especially when passing boolean literals as arguments to specify them as keyword parameters.